.env.example

```diff
@@ -1,16 +1,19 @@
-# Gemfile
-GEM_PG_VERSION=1.2.2
+# Database
+DB_HOST=postgis
+DB_NAME=gtt
+DB_USER=gtt
+DB_PASS=gtt
+DB_PORT=5432
 
-# docker-compose.yml - postgres
-POSTGRES_USER=gtt
-POSTGRES_PASSWORD=gtt
-POSTGRES_DB=gtt
-
-# config/configuration.yml
-SMTP_ENABLE_STARTTLS_AUTO=true # (or false)
-SMTP_ADDRESS=smtp.gmail.com
-SMTP_PORT=587
-SMTP_DOMAIN=smtp.gmail.com
-SMTP_AUTHENTICATION=plain # (or login)
+# Email
+SMTP_ADDRESS=smtp
+SMTP_DOMAIN=smtp.example.com
+SMTP_PORT=1025
 SMTP_USER_NAME=YOUR_EMAIL_ADDRESS
 SMTP_PASSWORD=YOUR_PASSWORD
+SMTP_AUTHENTICATION=plain # (or login)
+SMTP_ENABLE_STARTTLS_AUTO=false
```

Comment:
Please check the following comments.
- https://github.com/gtt-project/docker-gtt/pull/54/files#r971610242
- https://github.com/gtt-project/docker-gtt/pull/54/files#r971611381